libs/blocks/marketo/marketo.js

```diff
@@ -156,13 +119,16 @@ const init = (el) => {
         formWrapper.append(description);
       }
 
-      const marketoForm = createTag('form', { ID: `mktoForm_${formID}`, class: 'hide-errors' });
-      formWrapper.append(marketoForm);
-      fragment.append(error, formWrapper);
+      const marketoForm = createTag('form', { ID: `mktoForm_${formID}`, class: 'hide-errors', style: 'opacity:0;visibility:hidden;' });
+      const span1 = createTag('span', { id: 'mktoForms2BaseStyle', style: 'display:none;' });
```

Comment:
Just curious what these hidden `span1` and `span2` elements are for?

Reply:
Rob asked us to put these in, so they're helpers for code being loaded by the Marketo product